RecoLocalTracker/SiPixelRecHits/interface/pixelCPEforGPU.h

```diff
@@ -40,7 +44,11 @@ namespace pixelCPEforGPU {
 
     float x0, y0, z0;  // the vertex in the local coord of the detector
 
-    float sx[3], sy[3];  // the errors...
+    // float apeX, apexY;  // ape^2
+    uint8_t sx2, sy1, sy2;
+    uint8_t sigmax[16], sigmax1[16], sigmay[16];  // in micron
```

Comment:
what is `16` here ?

Reply:
the number of "bins"

Reply:
Is it obvious to people familiar with the CPE why 16 ?
I clearly have no idea :-)

Reply:
Absolutely not. It is just my decision to use 16 bins. I use GenError as a black box.